src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AttendanceService.kt

```diff
@@ -17,12 +17,12 @@ import javax.transaction.Transactional
 
 @Service
 class AttendanceService(
-    private val attendanceRepository: AttendanceRepository,
-    private val attendanceChangesRepository: AttendanceChangesRepository,
-    private val elite2ApiService: Elite2ApiService,
-    private val iepWarningService: IEPWarningService,
-    private val nomisEventOutcomeMapper: NomisEventOutcomeMapper,
-    private val telemetryClient: TelemetryClient) {
+        private val attendanceRepository: AttendanceRepository,
```

Comment:
not sure what's going on here - kotlin normally 2 character indent in DPS, but file seems to be 4?